trymito.io/pages/teams.tsx

```diff
@@ -0,0 +1,232 @@
+import Link from 'next/link';
+
+
+const Teams: NextPage = () => {
+
+  return (
+    <>
+      <Head>
+        <title>Mito | Teams </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
+      
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+
+            <section className={titleStyles.title_card + ' ' + titleStyles.grid_card}>
+                <h1 className={titleStyles.title}>
+                    Embrace Python by leveraging your team’s Excel mastery
+                </h1>
+                <p className={titleStyles.description}>
+                    Onboard your team to Mito in minutes
+                </p>
+                <div className={homeStyles.cta_button_and_video_spacer}>
+                    <CTAButtons variant='contact' />
+                </div>
+            </section>
+            <section>
+                <div className={pageStyles.subsection}>
+                    <div className={homeStyles.functionality_text}>
+                        <p>
+                            <b className='text-primary'>Spreadsheet-first teams </b> use Mito to transition to Python while preserving 
+                            the spreadsheet interface that we all know and love. 
+                        </p>
+                        <p> 
+                            <b className='text-primary'>Python-first teams </b> use Mito to write code faster without spending time searching Google or Stack Overflow.
+                        </p>
+                        <a href="mailto:founders@sagacollab.com" className={pageStyles.link_with_p_tag_margins}>
+                            Contact the Mito team →
+                        </a>
+                    </div>
+                    <div className={homeStyles.functionality_media}>
+                        <Image src={'/explore.png'} alt='Explore your data with Mito' width={500} height={250} layout='responsive'/>
+                    </div>
+                </div>
+            </section>
+            <section>
+                <div className={pageStyles.subsection + ' center'}>
+                    <h1>
+                        Deploy Mito in Jupyter in minutes
+                    </h1>
+                </div>
+                <div className={pageStyles.subsection}>
+                    <div className={securityStyles.security_bullet_container}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={JupyterIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Jupyter <br/> Compatible
+                        </h1>
+                        <p>
+                            Mito is compatible with Jupyter Hub, Jupyter Lab, and Jupyter Notebooks, so you don’t need to manage any new infrastructure.  
```

Comment:
Jupyter Hub, Jupyter Lab -> JupyterHub, JupyterLab

Reply:
- Mito is compatible with JupyterHub, JupyterLab, and Jupyter Notebooks. No need to manage new infrastructure or lock yourself into proprietary software.